src/Host/Client/Impl/Microsoft.R.Host.Client.csproj

```diff
@@ -198,6 +193,10 @@
       <Project>{8d408909-459f-4853-a36c-745118f99869}</Project>
       <Name>Microsoft.Common.Core</Name>
     </ProjectReference>
+    <ProjectReference Include="..\..\..\R\Core\Impl\Microsoft.R.Core.csproj">
+      <Project>{0c4bce1d-3cb8-4e2a-9252-58784d7f26a5}</Project>
+      <Name>Microsoft.R.Core</Name>
```

Comment:
Since Host.Client may become public API, the fewer assemblies the better. Also - R.Core is R parser, it has nothing to do with MRC or R Installation. MRC and R Install actually have more to do with the host...